### DISCUSSION TITLE
Set location of .env file

### DISCUSSION THREAD

**Discussion by danielbehrendt** [deployphp/deployer#2813] [Help needed]
Is it possible to have Deployer setting up the .env file one level up (above "current") without manually symlinking it into "current" folder? The reason is that we use ploi.io which gives access to .env file via its UI and it expects the .env file on the same level as current.

We use Deployer 7.

**Comment by antonmedv** [comment]
Why symlinking not working?

```php
task('env', function () {
    run('{{bin/symlink}} .env {{current_path}}/.env');
});
```

**Reply by danielbehrendt** [reply]
Solved it by:

`ln -sf {{deploy_path}}/.env {{current_path}}/.env`
